mesh_geometry_datagather_utils.py

```diff
@@ -158,25 +158,45 @@ def parse_obj_to_geometrydata(meshObj, parentSkeleton, shaderIndex, correctedNor
         uvlayer = bm.loops.layers.uv[0]
         uvlayer2 = bm.loops.layers.uv[1]
 
+    #vertex color layers (should be two but who knows what people are doing)
+    vcLayer = None
+    vcLayer2 = None
+
+    if len(bm.loops.layers.color) > 0:
+        vcLayer = bm.loops.layers.color[0]
+
+        if len(bm.loops.layers.color) > 1:
+            vcLayer2 = bm.loops.layers.color[1]
+
     #fill uvCoords and qtangents with blank entries so that we can fill them in any order
-    geom.uvCoords = [(0.0, 0.0)] * len(geom.vertPositions)
-    geom.uvCoords2 = [(0.0, 0.0)] * len(geom.vertPositions)
+    geom.uvCoords = [(1.0, -1.0)] * len(geom.vertPositions)
+    geom.uvCoords2 = [(1.0, -1.0)] * len(geom.vertPositions)
```

Comment:
Technically it should not make a difference. The uvs are just placed on one pixel and this pixel might have a different colour then before. I changed this back for now as it makes probably no difference but if it makes a visual difference it might be worth investigating the best solution. Maybe OpenIV devs decided for 1 -1 for some reason and your model not being red was acutally wrong ^^.